attributes/default.rb

```diff
@@ -90,21 +91,12 @@
 
 default['splunk']['splunk_servers'] = []
 
-case node['platform_family']
-when 'rhel', 'fedora', 'suse', 'amazon'
-  if node['kernel']['machine'] == 'x86_64'
-    default['splunk']['forwarder']['url'] = 'https://download.splunk.com/products/universalforwarder/releases/6.6.0/linux/splunkforwarder-6.6.0-1c4f3bbe1aea-linux-2.6-x86_64.rpm'
-    default['splunk']['server']['url'] = 'https://download.splunk.com/products/splunk/releases/6.6.0/linux/splunk-6.6.0-1c4f3bbe1aea-linux-2.6-x86_64.rpm'
-  else
-    default['splunk']['forwarder']['url'] = 'https://download.splunk.com/products/universalforwarder/releases/6.6.0/linux/splunkforwarder-6.6.0-1c4f3bbe1aea.i386.rpm'
-    default['splunk']['server']['url'] = 'https://download.splunk.com/products/splunk/releases/6.3.10/linux/splunk-6.3.10-75de5c491bd1.i386.rpm'
-  end
-when 'debian'
-  if node['kernel']['machine'] == 'x86_64'
-    default['splunk']['forwarder']['url'] = 'https://download.splunk.com/products/universalforwarder/releases/6.6.0/linux/splunkforwarder-6.6.0-1c4f3bbe1aea-linux-2.6-amd64.deb'
-    default['splunk']['server']['url'] = 'https://download.splunk.com/products/splunk/releases/6.6.0/linux/splunk-6.6.0-1c4f3bbe1aea-linux-2.6-amd64.deb'
-  else
-    default['splunk']['forwarder']['url'] = 'https://download.splunk.com/products/universalforwarder/releases/6.6.0/linux/splunkforwarder-6.6.0-1c4f3bbe1aea-linux-2.6-intel.deb'
-    default['splunk']['server']['url'] = 'https://download.splunk.com/products/splunk/releases/6.3.10/linux/splunk-6.3.10-75de5c491bd1-linux-2.6-intel.deb'
-  end
-end
+default['splunk']['forwarder']['url'] = value_for_platform_family(
+  %w(rhel fedora suse amazon) => 'https://download.splunk.com/products/universalforwarder/releases/6.6.0/linux/splunkforwarder-6.6.0-1c4f3bbe1aea-linux-2.6-x86_64.rpm',
```

Comment:
Was the removal of 32bit support intentional?

Reply:
well, yes.